ansible/roles/contiv/tasks/netplugin.yml

```diff
@@ -0,0 +1,81 @@
+- name: ensure pluginDir exists
+  file: path=/usr/libexec/kubernetes/kubelet-plugins/net/exec recurse=yes state=directory
+
+- name: copy contiv cni conf file
+  copy: src=contiv_cni.conf dest=/usr/libexec/kubernetes/kubelet-plugins/net/exec/contiv_cni.conf
+
+- name: ensure cni bin dir exists
+  file: path=/opt/cni/bin recurse=yes state=directory
+
+- name: copy contiv cni exec file
+  copy: src={{ contiv_bin_path }}/contivk8s dest=/opt/cni/bin/contivk8s.bin mode=0755
+
+- name: ensure contiv config directory exists
+  file: path=/opt/contiv/config recurse=yes state=directory
+
+- name: setup config for the contiv cni plugin
+  template: src=contiv.cfg.j2 dest=/opt/contiv/config/contiv.json
+
+- name: copy netplugin exec file
+  copy: src={{ contiv_bin_path }}/netplugin dest=/usr/bin/netplugin mode=0755
+
+- name: get openstack kilo rpm
+  get_url:
+    url: https://repos.fedorapeople.org/repos/openstack/openstack-kilo/rdo-release-kilo-1.noarch.rpm
+    dest: /tmp/rdo-release-kilo-1.noarch.rpm
+    validate_certs: False
+  environment:
+    http_proxy: "{{ http_proxy|default('') }}"
+    https_proxy: "{{ https_proxy|default('') }}"
+  when: ansible_os_family == "RedHat"
```

Comment:
The rest of the contrib project uses the ansible_distribution fact for OS detection. For example: ansible_distribution == "CentOS"

Can you use ansible_distribution == "CentOS" instead of ansible_os_family == "RedHat" so the Contiv role follows the same design pattern as the rest of the project?